draft-ietf-httpbis-origin-frame.md

```diff
@@ -101,49 +106,65 @@ REMOVE (0x2):
 The set of origins (as per {{!RFC6454}}) that a given connection might be used for is known in this
 specification as the Origin Set.
 
-When a connection is first established, its Origin Set is defined to be those origins that the client would normally consider the connection authoritative for; see {{!RFC7540}}, Section 10.1.
+When a connection is first established, its Origin Set is defined to be the origin composed from:
+  - Scheme: "https"
+  - Host: the value sent in Server Name Indication {{!RFC6066}}
+  - Port: the local port of the connection on the server
+
+If SNI is not sent, the Origin Set is empty when the connection is established.
```

Comment:
This is a retroactive change to h2.  It says that you can't coalesce.  It also says that you can't *use* a connection if you didn't use SNI.  I think that you need to include some weasel-words about how this doesn't really apply until the first ORIGIN frame is sent or received.